src/pages/Details/components/TotalInfo.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import '../components/TotalInfo.scss';
+import { useState } from 'react';
+import ProductDescription from './ProductDescription';
+import Review from './Review';
+import MoreInformation from './MoreInformation';
+import { Link } from 'react-router-dom';
+
+const TotalInfo = () => {
+  // const [productDetailClick, setproductDetailClick] = useState(
+  //   <ProductDescription />
+  // );
+
+  // const changer = e => {
+  //   e.preventDefault();
+  //   setproductDetailClick();
+  // };
+  const [state, setstate] = useState(0);
+
+  function tab0() {
+    setstate(0);
+  }
+  function tab1() {
+    setstate(1);
+  }
+  function tab2() {
+    setstate(2);
+  }
```

Comment:
```suggestion
  const [currentTab, setCurrentTab] = useState(0);

  const tabList = [
    { id: 0, title: '상품설명', component: <ProductDescription /> },
    { id: 1, title: '후기', component: <Review /> },
    { id: 2, title: '상세정보', component: <MoreInformation /> },
  ];
```
해당 부분은 위의 있는 코드를 활용하면 조금 더 효율적으로 구현할 수 있습니다!
이 부분은 라운지 시간때 조금 더 자세하게 설명 드릴게요!